README.md

```diff
@@ -2,11 +2,12 @@
 
 [![NPM](https://nodei.co/npm/minio.png)](https://nodei.co/npm/minio/)
 
-The MinIO JavaScript Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+The MinIO JavaScript Client SDK provides high level APIs to access any Amazon S3 compatible object storage server.
 
-This quickstart guide will show you how to install the client SDK and execute an example JavaScript program. For a complete list of APIs and examples, please take a look at the [JavaScript Client API Reference](https://min.io/docs/minio/linux/developers/javascript/API.html) documentation.
+This guide will show you how to install the client SDK and execute an example JavaScript program. 
+For a complete list of APIs and examples, please take a look at the [JavaScript Client API Reference](https://min.io/docs/minio/linux/developers/javascript/API.html) documentation.
 
-This document assumes that you have a working [nodejs](http://nodejs.org/) setup in place.
+This document presumes you have a working [Node.js](http://nodejs.org/) development environment, version 16 or later.
```

Comment:
Is this version requirement accurate?

Reply:
Although we support v12 as @trim21 says, I'd vote to state the last 2 LTS versions. So that means v16 (go out of LTS in August 2023), v18 (maintenance), v20 (current LTS)